README.md

```diff
@@ -89,46 +103,28 @@ An example demonstrating detailed usage.
 
 Note that following this guide does not prevent you from being disputed or guaruntee reporters will properly put a value on-chain. Tellor is decentralized.  This repo is a start to the education necessary for a fully decentralized oracle, but please focus on communication and working with reporters to prevent unneccesary disputes and at the same time encourage monitoring and punishment of bad data. 
 
-## Why JSON? 
-
-JSON's a well known standard, but we ultimately chose it for three reasons: 
-
-1.  Human Readable/ Understandable
-
-2.  Machine Readable  
-
-3.  Programming Language Agnostic
-
 
 # Query Descriptor Specifications
-All query descriptors shall comply with the following specifications:
+All query descriptors should be found in the dataSpecs.json file and shall comply with the following specifications:
 
-1. The top level descriptor string shall be a JSON `object` that defines a set of `name`/`value` pairs.
+1. The top level descriptor string shall be a JSON `object` that defines a set of `type` => `inputs` 
 
-2.  An object begins with `{`  and ends with `}`. Each name is followed by `:` colon and the name/value pairs are separated by a comma (`,`).  
+2.  The `type` should be a valid `QueryType`
 
-3. There shall be **no whitespace before or after the colon or comma separators**.
+3. The remaining `inputs` shall be query `parameters`, provided in the specified order.
 
-4. The top level object is ordered so that `name`/`value` pairs must be provided in the specified order.
-
-5. The first `name`/`value` pair shall provide the Query Type
-   - The `name` shall be the word `"type"`.  
-   - The `value` shall be the name of a valid `QueryType`
-
-6. The remaining `name`/`value` pairs shall be query `parameters`, provided in the specified order.
-
-7. There shall be no optional query parameters.  All query parameters shall be specified in the descriptor.
+4. There shall be no optional query parameters.  All query parameters shall be specified in the descriptor.
 
 ## Contract Interfaces
 
 Interacting with the TellorX contract (e.g. calls to  `tipQuery`, `submitValue`) requires the computation of two values: `_queryData`, and `_queryID`.  
 
 Both values are easily computed from the QueryType Descriptor JSON string.
-
+ 
 
 ### QueryData
 
-The `bytes` value of `_queryData` in contract calls shall be the UTF-8 encoded version of the query `descriptor` string.
+The `bytes` value of `_queryData` in contract calls shall be the UTF-8 encoded version of the parameters of the query `descriptor` string.
```

Comment:
Suggestion:
The `bytes` value of `_queryData` in contract calls shall be the ABI encoded query, as specified by the ABI descriptor.